tactics/hints.ml

```diff
@@ -1332,17 +1330,19 @@ let discharge_autohint obj =
 
 let hint_cat = create_category "hints"
 
-let inAutoHint : hint_obj -> obj =
-  declare_object
+let (objConstant : (Id.t * hint_obj) Libobject.Dyn.tag) =
+  declare_named_object_full
```

Comment:
you don't need _full if you immediately use Easy.inj
(non-full is the composition of full and Easy.inj)